src/app.js

```diff
@@ -13,10 +13,7 @@ const app = express()
 
 app.use(
     cors({
-        origin: [
-            "https://dnvr-zero-be.vercel.app/task",
-            "http://localhost:3000",
-        ],
+        origin: ["https://dnvr-zero-be.vercel.app", "http://localhost:3000"],
```

Comment:
I just removed the `/task` from this line, which is just some old code that needed to be cleaned up